ansible/roles/plpsql/templates/recovery.conf

```diff
@@ -1,4 +1,4 @@
 standby_mode = 'on'
 primary_slot_name = 'amsmetadb_3'
-primary_conninfo = 'host=hkgmetadb.infra.ooni.io port=5432 user=amsrepl password={{ amsrepl_hkgmetadb_password }}'
+primary_conninfo = 'host=hkgmetadb.infra.ooni.io port=5432 user=amsrepl sslcert=/srv/pl-psql/pg_user_amsrepl.cert sslcert=/srv/pl-psql/ssl/pg_user_amsrepl.key'
 trigger_file = '/srv/pl-psql/trigger.flag'
```

Comment:
You may want to add `sslmode=verify-full` here.
[_By default, PostgreSQL will not perform any verification of the server certificate._](https://www.postgresql.org/docs/9.6/libpq-ssl.html)

Reply:
Good point.